app/models/post/view.rb

```diff
@@ -13,7 +13,7 @@ def mark_favorite_read
     favorited_users = user.favorites.where(favorite: post.joined_authors).exists?
     return unless favorited_continuity || favorited_users
 
-    message = NotifyFollowersOfNewPostJob.notification_about(post, user, unread_only: true)
+    message = NotifyFollowersOfNewPostJob.notification_about(post, user, unreadonly: true)
```

Comment:
Is this on purpose?

Reply:
...I thought I fixed that. no, it's not on purpose.